packages/app/src/components/editor/ArticleWizard.tsx

```
@@ -438,61 +547,181 @@ export const ArticleWizard: React.FC<Props> = ({ draftId, articleId, onDone, onC
+              <img
+                src={coverImage}
+                alt="cover"
+                className="w-full h-40 object-cover rounded mt-2"
+              />
+            )}
+            <h2 className="mt-2 text-2xl font-heading">
+              {title || "Untitled"}
+            </h2>
+            <div
+              className="prose max-w-none whitespace-pre-wrap mt-2"
+              dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)